docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,271 @@
+# Getting Started With Soci Snapshotter
+This document is meant to be a guide to walk through an example of using the
+soci-snapshotter.  To get everything running we will need to do a few things : 
+- prerequistes
+- start a local oras registry
+- push an image using ctr to our registry
+- create and push a soci index to our registry
+- run a lazily loaded container from the registry
+
+Along the way we will make sure to inspect each of our actions to ensure that
+they worked.  This is valuable to help learn about the tool and debugging.
+
+## Prerequistes
+**docker** - to confirm that you have docker working please check with `docker
+version` and `docker images` - both should succeed.
+
+**containerd** - to confirm that you have containerd working please check with
+`ctr version` and `ctr i ls`
+
+**ctr** - to confirm that you have ctr, the containerd cli installed, check with
+`ctr version` and `ctr i ls`
+
+**soci-snapshotter** - this you will need to make sure you can build from
+source. You should be able to run `make` in the soci-snapshotter dir.
+Some common issues : 
+- getting the github repository `git clone
+https://github.com/awslabs/soci-snapshotter.git`
+- having the zlib developer tools (we use zlib to compress and read ztocs).
+Often fixed with `sudo yum install zlib-devel`
+- having the proper go version installed, as of this document please use 1.18+
+
+## Start a local ORA registry
```

Comment:
typo "ORA" vs. "ORAS" but also, per the discussion above, maybe use another name here. I think the real issue to start this section with is not the handling of arbitrary documents (most registries now support those), but the "referrers" feature support that will be needed in any OCI-compliant registry to handle the ztoc artifact reference. Currently the ORAS project fork of "distribution/distribution" has PRs which enable this.